modules/xmpp/JingleSessionPC.js

```diff
@@ -2277,6 +2277,20 @@ export default class JingleSessionPC extends JingleSession {
         });
     }
 
+    /**
+     * Resumes or suspends media transfer over the underlying peer connection.
+     *
+     * @param {boolean} enable
+     * @returns {Promise}
+     */
+    resumeOrSuspendMedia(enable) {
+        return this.peerconnection.tpcUtils.resumeOrSuspendMedia(enable)
+            .then(() => {
```

Comment:
What if it fails?

Reply:
It logs an error at the JitsiConference level - [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L3308)  or [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L3437)